app/views/AgentForTraderCheckRegisteredDetailsView.scala.html

```diff
@@ -134,10 +132,10 @@
     ).withHint(Hint(content = HtmlContent(paraContent)))
 )
 
-@searchLink
+    @paragraph(content = Html(messages("traderDetails.common.searchAgain", searchAgainLink)))
```

Comment:
Is the call to searchAgainLink able to find traderDetails.common.searchAgain.link automatically?

Reply:
Ah ok, thanks for the explanation.